examples/js/postprocessing/EffectComposer.js

```diff
@@ -127,12 +132,8 @@ THREE.EffectComposer.prototype = {
 
 	setSize: function ( width, height ) {
```

Comment:
As here: https://github.com/mrdoob/three.js/pull/6820/files#diff-f69be6f1b9bed3c3d7387ed929cd29faL13
But not done on resize here: https://github.com/mrdoob/three.js/pull/6820/files#diff-76f301411c5013d33c69fa37efb52263R146